public/js/components/AtomEdit/CustomEditors/TimelineFields/TimelineItem.js

```diff
@@ -24,9 +24,9 @@ export class TimelineItem extends React.Component {
 
   render () {
     const value = this.props.fieldValue || {
-      title: "-",
+      title: " ",
```

Comment:
I'm not sure this is valid in Dynamo, which is why a dash was used.

Reply:
Gaaaaaaaah 😱 Thanks for spotting. I'll take a look.

Reply:
So from messing around with the `CODE` & `DEV` DynamoDB tables it seems to accept the spaces. Do you know where this was covered in a PR or anywhere else?

Reply:
@susiecoleman and @philmcmahon Any idea? ^

Reply:
Dynamo doesn't allow an empty string...not sure if a space will make it in

Reply:
It seems to work from modifying this atom directly in Dynamo: https://atomworkshop.code.dev-gutools.co.uk/atoms/timeline/07a75b03-92fd-4c78-9052-ccbb25c478d4/edit

What do you think @shaundillon ?

Reply:
Ahh, that's what I was thinking. No idea why a dash was chosen over a space then. Maybe for visibility?

Reply:
💯 much better than the dash!